source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
Is it possible to exclude libraries from the server side rendering? Mixpanel is not happy